doc/source/_static/theme_overrides.css

```diff
@@ -0,0 +1,5 @@
+
+
+.sd-card-title {
+    color: var(--pst-color-link);
+}
```

Comment:
I am looking for a selector that only matches if the card contains a link. Unfortunately the link isn't in the parent hierarchy but included as a neighboring element of `sd-card-title` parent. Maybe something like [Adjacent sibling Combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator)? But I'm looking to match the first and not the second element. Does somebody know CSS well enough to help?

Reply:
@lagru In case you're still looking... [the `:has()` selector](https://css-tricks.com/the-css-has-selector/):

```css
/* Select only the .sd-card-title children of .sd-card
 * which also contain an `a` element.
 */
.sd-card:has(a) .sd-card-title {
    color: var(--pst-color-link);
}
```

Its argument handling is pretty robust, too...

```css
/* Select only the .sd-card-title children of .sd-card
 * which contains an external link.
 */
.sd-card:has(a.external) .sd-card-title {
    color: var(--pst-color-link);
}
```


[Support](https://caniuse.com/css-has) is... still not "widespread", but improving fast.

* Chrome/Edge since 105 (2022-08)
* Android since 117 (2023-09)
* Safari (both) since 15.4 (2022-03)
* Firefox **STILL** hidden behind a manual configuration opt-in :angry: 
* IE never :roll_eyes: 
* Opera since 91 (2022-09) (mobile since 73) (2023-01)
* Samsung Internet since 20 (2023-02).